### PR TITLE
Added Sentry for aggregating and tracking errors and outages (CU-32a5wb)

### PR DESCRIPTION
**Pull request by mariocimet**
Up on chatbot-dev.brave.coop, you can track errors here:https://sentry.io/organizations/brave-technology-coop/issues/?project=5743498

one way to see it in action is to just `curl --data "bla=1" https://chatbot-dev.brave.coop/`

### CODE REVIEW

**Review by trisapeace — APPROVED** [review]

File: server/.env.example [hunk 1]
```diff
@@ -63,3 +63,12 @@ PG_PORT_TEST=12345
 # Incoming API key used by Flic Buttons (we made this up, value can be found in 1password)
 FLIC_BUTTON_PRESS_API_KEY=testFlicApiKey
 FLIC_BUTTON_PRESS_API_KEY_TEST=testFlicApiKey # must be testFlicApiKey
+
+# Sentry data source name
+SENTRY_DSN=exampleSentryUrl.com
+
+# Environment tag for Sentry
+ENIRONMENT=exampleenvironment
```
[the remark this answers is not in the document]
Comment (mariocimet) [reply server/.env.example]: fixed.

**Review by trisapeace — APPROVED** [review]
Cool. Re-approved.

**Review by schwarrrtz — CHANGES_REQUESTED** [review]
@mariocimet it looks like you added a `package-lock.json` in the root directory of the repo, this is probably a mistake?

**Comment by schwarrrtz** [comment]
I ran `npm ci` and then ran the tests locally (using a local Postgres db) and everything passes for me. Once that extra file is removed I'd say this is good to go.

**Comment by mariocimet** [comment]
> @mariocimet it looks like you added a `package-lock.json` in the root directory of the repo, this is probably a mistake? Otherwise this looks great, I'm just finishing up reviewing the changes

done!

**Review by schwarrrtz — APPROVED** [review]

**Review by trisapeace — APPROVED** [review]

**Pull request merged**